teps/0138-decouple-api-and-feature-versioning.md

```diff
@@ -166,11 +166,11 @@ See [implementation plan](#implementation-plan) for more details on the PerFeatu
 
 All **new** features can only be enabled via per-feature flags. When they first get introduced as `alpha`, they will be disabled by default. When new features get promoted to `stable`, they will be enabled by default according to the following table:
 
-| Feature stability level | Default                     |
-|-------------------------|-----------------------------|
-| Stable                  | Enabled; Cannot be disabled |
-| Beta                    | Disabled                    |
-| Alpha                   | Disabled                    |
+| Feature stability level | Default  |
+| ----------------------- | -------- |
+| Stable                  | Enabled; Cannot be disabled once the flag is removed (after deprecation)  |
+| Beta                    | Disabled |
```

Comment:
Current docs says beta feature are enabled by default, we are going to change it right?
https://github.com/tektoncd/pipeline/blob/8123d6332245008983aabe1dc910a63bd2c6f261/api_compatibility_policy.md?plain=1#L120

Reply:
I think that we are not. Some of the discussion at: https://github.com/tektoncd/pipeline/issues/6948#issuecomment-1775574610. The table is for new features only. And the existing beta features guarded by `enable-api-fields` will still be turned on by default. 

I intend to clarify so at the doc updates via https://github.com/tektoncd/pipeline/pull/7090 (which is currently blocked by https://github.com/tektoncd/pipeline/pull/6941).